storage/tianmu/core/temp_table.h

```diff
@@ -111,12 +111,12 @@ class TempTable : public JustATable {
                                                                       // values (incl. null, if flag set)
     uint64_t ExactDistinctVals([[maybe_unused]] Filter *f) override { return common::NULL_VALUE_64; }
     bool IsDistinct([[maybe_unused]] Filter *f) override { return false; }
-    size_t MaxStringSize(Filter *f = nullptr) override;  // maximal byte string length in column
-    int64_t RoughMin(Filter *f = nullptr, common::RSValue *rf = nullptr) override {
+    size_t MaxStringSize([[maybe_unused]] Filter *f = nullptr) override;  // maximal byte string length in column
+    int64_t RoughMin([[maybe_unused]] Filter *f = nullptr, [[maybe_unused]] common::RSValue *rf = nullptr) override {
```

Comment:
although your way is right, but we recommend following the conventional usage as below.
```
//innodb handler definitions.
  FT_INFO *ft_init_ext_with_hints(uint inx [[maybe_unused]],
                                  String *key [[maybe_unused]],
                                  Ft_hints *hints [[maybe_unused]]) override {
    ut_d(ut_error);
    ut_o(return (nullptr));
  }

  int ft_read(uchar *buf [[maybe_unused]]) override {
    ut_d(ut_error);
    ut_o(return (HA_ERR_WRONG_COMMAND));
  }
```

Reply:
fixed it.  
I will follow this conventional usage.